ansible-ipi-install/roles/installer/tasks/07_deploy_ocp.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: IPI-INSTALL Deploy OpenShift Cluster
```

Comment:
Remove IPI-INSTALL so that all task names follow a similar pattern (better for readability imho)